parachain/runtime/snowbridge/src/lib.rs

```diff
@@ -357,6 +360,49 @@ impl cumulus_pallet_xcm_handler::Config for Runtime {
 	type AccountIdConverter = LocationConverter;
 }
 
+// Governance
+
+impl pallet_sudo::Config for Runtime {
+	type Event = Event;
+	type Call = Call;
+}
+
+type EnsureRootOrHalfLocalCouncil = EnsureOneOf<
```

Comment:
so this basically ensures either sudo (ie, alice on local/rococo) or one half of council, right?

are all 3 lines part of this check?

Reply:
yeah, exactly. Either sudo or at least half of council. We can always fine-tune the proportion later.